programmers/로또.js

```diff
@@ -0,0 +1,21 @@
+function solution(lottos, winNums) {
+    
+    let [zCnt, wCnt] = [0, 0];
```

Comment:
```suggestion
    let [zeroCount, count] = [0, 0];
```
변수명 잘 지어주셨는데 좀 더 명확하면 더 좋을 것 같아요